ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/WebSocketService.java

```diff
@@ -91,10 +91,8 @@ private WebSocketService(
                 new HttpServerOptions()
                     .setHost(configuration.getHost())
                     .setPort(configuration.getPort())
-                    .setHandle100ContinueAutomatically(true)
-                    .setCompressionSupported(true)
-                    .addWebSocketSubProtocol("undefined"))
-            .webSocketHandler(websocketHandler())
+                    .setWebsocketSubProtocols("undefined"))
```

Comment:
What are these options that were removed?

Reply:
this was not meant to be changed...

Reply:
reverted